### PR TITLE
Integrate 5.6.0

### PR DESCRIPTION
**Pull request by carlescufi**

### CODE REVIEW

**Comment by ioannisg** [comment]
Do you need a Zephyr PR to check integration with .west?

**Comment by carlescufi** [comment]
> Do you need a Zephyr PR to check integration with .west?

https://github.com/zephyrproject-rtos/zephyr/pull/23610

**Comment by carlescufi** [comment]
> I think Core_R should be a follow up commit since its not part of the upstream CMSIS_5 right now.
@galak 
Done

File: CMSIS/Core_R/Include/core_cr7.h [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+/* Configuration of the Cortex-R7 Processor and Core Peripherals */
+#ifndef __CR_REV
+#define __CR_REV                0U
+#endif
+
+#ifndef __FPU_PRESENT
+#define __FPU_PRESENT           0U
+#endif
+
+#define __GIC_PRESENT           1U
```
Comment (Najon) [inline CMSIS/Core_R/Include/core_cr7.h:50]: Hi, while trying to add support for Cortex R7 in Zephyr, I had to disable __GIC_PRESENT and __TIM_PRESENT.
Zephyr already have support for GIC, so we should not use both drivers.

What should be the recommended way ?
Disabling GIC in CMSIS or in Zephyr.
Thanks

**Pull request merged**